config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -12,6 +12,18 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-release-publishing-bot
       testgrid-tab-name: build
+  - name: pull-publishing-bot-test
+    always_run: true
+    decorate: true
+    path_alias: k8s.io/publishing-bot
+    spec:
+      containers:
+      - image: golang:1.13
```

Comment:
I think we should be able to update this to 1.16 for publishing-bot (all prowjobs)

Reply:
ah, just saw https://github.com/kubernetes/publishing-bot/issues/248 :smile:

Reply:
should we do it in a follow-up or in this PR?

Reply:
should we be fine to do it in this PR IMO

Reply:
will do thanks!

Reply:
update to go1.16, ptal @nikhita